Makefile

```diff
@@ -45,20 +48,33 @@ assemble:
 	cp -r $(HERE)/zq2 $(BINDIR)
 
 IMAGE_TAG ?= developer-portal:latest
+IMAGE_TAG_NOZQ2 ?= developer-portal-nozq2:latest
+ZQ2 ?= 1
 
 build:
-	docker buildx build --build-arg VERSION="${VERSION}" . -t $(IMAGE_TAG)
+	docker buildx build --build-arg ZQ2=1 --build-arg VERSION="${VERSION}" -f Dockerfile . -t $(IMAGE_TAG)
```

Comment:
Yes. For local testing, I want to build both images:

 - Because remembering which one you configured last time locally is hard.
 - To make sure that they both build and one doesn't bitrot